File: Remove Duplicates from Sorted Array II - 80/ans.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        if (nums.size() == 0)
            return 0;
        int c = 1;
        int location = 1;
        for (int i = 1; i < nums.size(); ++i){
            if (nums[i] == nums[i - 1])
                c++;
            else
                c = 1;
            if (c <= 2){
                nums[location++] = nums[i];
            }
        }
        return location;
    }
};

File: Binary Search - 704/ans.kt
class Solution {
    fun search(nums: IntArray, target: Int): Int {
        var high = nums.size - 1
        var low = 0
        while (low <= high){
            var mid = low + (high - low) / 2
            
            if (nums[mid] == target)
                return mid
            if (nums[mid] < target)
                low = mid + 1
            else
                high = mid - 1
        }
        return -1
    }
}
